draft-ietf-lamps-kyber-certificates.md

```diff
@@ -253,20 +242,8 @@ The fields in SubjectPublicKeyInfo have the following meaning:
   algorithms defined in this document always encode the public key
   as TODO pick format e.g., exact multiple of 8 bits?.
 
-The following is an example of a ML-KEM-512 public key encoded using the
-textual encoding defined in {{?RFC7468}}:
-
-<aside markdown="block">
-  WARNING: This example was generated prior to NIST
-  finalizing [DRAFTFIPS203].
-</aside>
-
-~~~
-  -----BEGIN PUBLIC KEY-----
-  TODO insert example public key
-  -----END PUBLIC KEY-------
-~~~
-
+{example-public} contains an example of an id-alg-ml-kem-768 public key
```

Comment:
Appendix says it's ML-KEM-512, here says 768.  Which one?  No actual examples to verify this against :)

Reply:
Yep should be 768!